addons/web/static/src/scss/fields.scss

```diff
@@ -363,6 +363,7 @@
             }
 
             > .fa {
+                padding: 4px;
```

Comment:
The commit message explains something that this commit does not do.

Reply:
Done, changed the commit message.